handler.go

```diff
@@ -447,10 +454,11 @@ func (fm *Frontman) FetchInput(inputFilePath string) (*Input, error) {
 		if fm.Config.HubUser != "" {
 			// it may be useful to log the Hub User that was used to do a HTTP Basic Auth
 			// e.g. in case of '401 Unauthorized' user can see the corresponding user in the logs
-			return nil, fmt.Errorf("InputFromHub(%s:***): %s", fm.Config.HubUser, err.Error())
+			err = errors.Wrapf(err, "InputFromHub(%s:***) error", fm.Config.HubUser)
+			return nil, err
 		}
-
-		return nil, fmt.Errorf("InputFromHub: %s", err.Error())
+		err = errors.Wrap(err, "InputFromHub error")
+		return nil, err
```

Comment:
errors.Wrapf

Reply:
We don't need Wrapf here, because no additional context beside error.

Reply:
meant changing
```
err = errors.Wrap(err, "InputFromHub error")
return nil, err
```

to
```
return nil, errors.Wrap(err, "InputFromHub error")
```

Reply:
It's the style that I won't change, extremely hard to track function calls in return arguments

Reply:
Fixed